test/system/030-run.bats

```diff
@@ -1256,6 +1249,17 @@ EOF
 
     run_podman image unmount $IMAGE
 
+    # the TMPDIR must be accessible by different users as the following tests use different mappings
+    chmod 755 $PODMAN_TMPDIR
+
+    # check if the underlying file system supports idmapped mounts
+    run_podman '?' run --rm --uidmap=0:1000:10000 --rootfs $romount:idmap true
+    if [[ $status -ne 0 ]]; then
+        if [[ "$output" =~ "failed to create idmapped mount: invalid argument" ]]; then
+            skip "idmapped mounts not supported"
+        fi
```

Comment:
In the version I was just working on, I add:
```
        # Any other error is fatal
        die "Cannot create idmap mount: $output"
```
...which I think is important. Otherwise, our versions are identical, and I'm OK with your commit sequence.

Reply:
thanks, amended and pushed a new version